app/src/main/java/com/example/android/tvleanback/ui/PlaybackOverlayFragment.java

```diff
@@ -109,10 +109,15 @@
     private Timer mClickTrackingTimer;
     private int mClickCount;
 
+    private MediaController mMediaController;
+    private android.media.session.MediaController.Callback mMediaControllerCallback = new MediaControllerCallback();
```

Comment:
Line too long. Wrap on two lines, or possibly shorten type to MediaController.Callback.